docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,302 @@
+# Version 1.0.0
+
+TODO: Discord / YouTube / Twitter
+
+Today is finally the day... AutoGluon 1.0 has arrived!! After [over four years of development](https://automlpodcast.com/episode/autogluon-the-story) and [2052 commits from 111 contributors](https://github.com/autogluon/autogluon/graphs/contributors),
+we are excited to share with you the culmination of our efforts to create and democratize the most powerful, easy to use, and feature rich automated machine learning system in the world.
+AutoGluon 1.0 comes with transformative enhancements to predictive quality resulting from the combination of multiple novel ensembling innovations, spotlighted below.
+Besides performance enhancements, many other improvements have been made that are detailed in the individual module sections.
+
+Note: Loading models trained on older versions of AutoGluon is not supported. Please re-train models using AutoGluon 1.0.
+
+This release contains XXX commits from 17 contributors!
+
+Special thanks to @LennartPurucker for leading development of dynamic stacking, @geoalgo for co-authoring [TabRepo](https://github.com/autogluon/tabrepo) to enable Zeroshot-HPO, @ddelange for helping to add Python 3.11 support, and @mlgowacki100 for providing numerous comments and suggestions
```

Comment:
I would put a period at the end of this sentence.

Reply:
added